Cargo.toml

```diff
@@ -29,15 +29,8 @@ hashes = ["bitcoin_hashes", "secp256k1/bitcoin_hashes"]
 use-serde = ["serde", "secp256k1/serde"]
 use-rand = ["rand", "secp256k1/rand"]
 
-# Use this feature to not compile the bundled libsecp256k1 C symbols,
-# but use external ones. Use this only if you know what you are doing!
-external-symbols = ["secp256k1-zkp-sys/external-symbols", "secp256k1/external-symbols"]
-
-# Do not use this feature! HAZMAT. (meant for Fuzzing only. this is *BROKEN CRYPTOGRAPHY*)
-fuzztarget = ["secp256k1-zkp-sys/fuzztarget", "secp256k1/fuzztarget"]
-
 [dependencies]
-secp256k1 = { git = "https://github.com/rust-bitcoin/rust-secp256k1" }
+secp256k1 = "0.20.0"
```

Comment:
:question: Is it important to depend on the specific patch version?

Reply:
Nope, it is actually useless because cargo doesn't use it by default :)